### DISCUSSION TITLE
Report

### DISCUSSION THREAD

**Discussion by venteo22** [TheAlbertDev/MASB#70] [Q&A] [answered]
Hello Albert,
Regarding to the report, we don't know how to do it with GitHub and where we have to upload the file.

**Comment by TheAlbertDev** [comment, marked as answer]
Hello @venteo22,

As the script indicates in the deliverables section, you have to make the report in the same folder where the `README.md` file is located. For session 1, you have to create a file called `REPORT.md` in the `arduino/` folder (which will be inside the folder where you have cloned the repository in your computer).

To edit the file, you can use any of the applications recommended in the deliverables section. I used Typora to write the scripts and it was free, that's why it appears as recommended. Now it is paid. You have another very good alternative and free option which is Visual Studio Code, which allows you to write markdown and preview the document at the same time.

Finally, you generate the document and modify it in your local repository (that is, in the folder on your computer where you have made the clone). Once you have finished it, upload it to the remote repository (GitHub) using git. That is, you do a

```bash
git add arduino/REPORT.md
git commit -m "Report finished"
git push
```

As it's a question, I move the topic to the Q&A section. For future occasions, ask the questions to the community and not to me directly since maybe a colleague can help you, but doesn't answer you because they sees that the question is directly addressed to me. And if for whatever reason, the question is addressed to someone specific, refer to the recipient by their username as I've done in this answer so that the recipient receives a notification.

I hope I've been able to answer your question. If so, please mark the answer as a solution. If not, feel free to continue the discussion until it's solved.

And don't forget to introduce yourself in the Introductions!
